<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" >
     <item>
      <widget class="QListWidget" name="styleListWidget" >
       <property name="alternatingRowColors" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4" >
       <item>
        <widget class="QGroupBox" name="stylePreviewer" >
         <property name="title" >
          <string>Preview</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4" >
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3" >
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout" >
              <item>
               <widget class="QComboBox" name="comboBox" >
                <item>
                 <property name="text" >
                  <string>New Item</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>New Item</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>New Item</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>New Item</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>New Item</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>New Item</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>New Item</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="toolButton" >
                <property name="text" >
                 <string>...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2" >
              <item>
               <widget class="QGroupBox" name="groupBox" >
                <property name="title" >
                 <string>GroupBox</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout" >
                 <item>
                  <widget class="QRadioButton" name="radioButton_2" >
                   <property name="text" >
                    <string>RadioButton</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="radioButton" >
                   <property name="text" >
                    <string>RadioButton</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="Line" name="line" >
                   <property name="orientation" >
                    <enum>Qt::Horizontal</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="checkBox" >
                   <property name="text" >
                    <string>CheckBox</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_2" >
                <item>
                 <widget class="QPushButton" name="pushButton" >
                  <property name="text" >
                   <string>PushButton</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer" >
                  <property name="orientation" >
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0" >
                   <size>
                    <width>20</width>
                    <height>28</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QProgressBar" name="progressBar" >
                  <property name="value" >
                   <number>24</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3" >
              <item>
               <widget class="QSlider" name="horizontalSlider" >
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox" />
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QScrollBar" name="verticalScrollBar" >
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>64</width>
           <height>38</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRestore" >
       <property name="text" >
        <string>Revert</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonApply" >
       <property name="text" >
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
